src/lib/db.ts

```diff
@@ -757,7 +757,8 @@ export class DB {
 
   private monitorWorkspaceVisibility = (workspace: WorkspaceModelType) => {
     if (this.workspaceModelDisposers[workspace.sectionId]) {
-      this.workspaceModelDisposers[workspace.sectionId]();
+      // Workspaces ignores any duplicate workspaces created for a sectionId, so don't listen to them
+      return;
```

Comment:
@dougmartin This is the issue I mentioned on your PR about duplicate workspaces & needing to ignore the second one here